src/macros.jl

```diff
@@ -252,18 +252,12 @@ end
 #     LinearConstraint(aff, lb-offset, ub-offset)
 # end
 
-constructconstraint!(quad::QuadExpr, sense::Symbol) = QuadConstraint(quad, sense)
+constructconstraint!(quad::QuadExpr, set::Symbol) = QuadConstraint(quad, sense)
```

Comment:
Oops, I didn't want to change this method, I am not using it so it is for a further PR